models/training-tuning-scripts/root-cause-models/requirements.txt

```diff
@@ -0,0 +1,8 @@
+cudf==0+untagged.1.g788bddd
```

Comment:
should this be same cudf version used by morpheus?

Reply:
upgraded the versions